package.json

```diff
@@ -27,6 +27,7 @@
     "ethereum-cryptography": "^1.0.1",
     "ethereumjs-util": "^7.0.1",
     "ethers": "^5.7.1",
+    "hardhat-tracer": "^1.2.1",
```

Comment:
I saw you added this in another PR you're working on please add this as a separate PR, we merge it to nexus-v2 and avoid conflicts

Reply:
Ok I made another [PR](https://github.com/NexusMutual/smart-contracts/pull/763) for it :ok_hand: